main/webapp/modules/core/scripts/project/browsing-engine.js

```diff
@@ -234,6 +242,22 @@ BrowsingEngine.prototype.removeFacet = function(facet) {
   }
 };
 
+BrowsingEngine.prototype.hideShowBrowsingFacetSplash = function(clicked) {
+  var hideBrowsingFacetSplash = JSON.parse(Refine.getPreference("ui.browsing.hideBrowsingFacetSplash", false));
+  var checkbox = $('#browsing-panel-checkbox');
+
+  if(checkbox.prop('checked')) {
+    Refine.setPreference('ui.browsing.hideBrowsingFacetSplash', true);
+  }
+
+  if(hideBrowsingFacetSplash === true || clicked === true) {
+    $('.browsing-panel-help').addClass('hideFacetSplash');
+  }
+  else {
+    $('.browsing-panel-help').removeClass('hideFacetSplash');
+   }
```

Comment:
```suggestion
  } else {
    $('.browsing-panel-help').removeClass('hideFacetSplash');
  }
```
Just some spacing and indentation.